src/Features.php

```diff
@@ -90,6 +92,15 @@ private static function get_fields(): array {
 				'id'   => self::THEME_EDITOR,
 				'type' => 'checkbox',
 			],
+			[
+				'name' => __( 'Use beta blocks in posts', 'planet4-master-theme-backend' ),
```

Comment:
Isn't this formulation a bit confusing since we have posts, pages and campaigns? 🤔